docs/epoxy.md

```diff
@@ -0,0 +1,91 @@
+# Epoxy
+
+[Epoxy](https://github.com/airbnb/epoxy) is an open source library by Airbnb that makes it easier to build complex  layouts. It builds on top of RecyclerView, and allows you to work with multiple view types in a declarative, reactive way.
```

Comment:
maybe up here we can stress that Epoxy can optionally be used with MvRx as the view layer